### PR TITLE
feat: update snap label to include "Beta" keyword

### PR DESCRIPTION
**Pull request by gantunesr**
## **Description**

Update the snap account label from `Snaps` to `Snaps (Beta)`. 

## **Manual testing steps**

1. Install the [Simple Snap Keyring](https://metamask.github.io/snap-simple-keyring/latest/)
2. Create a new snap account
3. Notice the label says `Snaps (Beta)`

## **Screenshots/Recordings**

_[screenshot]_

Screenshot 2023-10-06 at 00 41 01

## **Related issues**

Fixes https://github.com/MetaMask/accounts-planning/issues/88

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained:
  - [x] What problem this PR is solving.
  - [x] How this problem was solved.
  - [x] How reviewers can test my changes.
- [x] I’ve indicated what issue this PR is linked to: Fixes #???
- [x] I’ve included tests if applicable.
- [ ] I’ve documented any added code.
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)).
- [x] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/pages/home/beta/beta-home-footer.component.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ const BetaHomeFooter = () => {
       </a>{' '}
       |{' '}
       <a href={SUPPORT_LINK} target="_blank" rel="noopener noreferrer">
-        {t('needHelpFeedback')}
+        {t('needHelpFeedback')} ajnddwjn
```
Comment (montelaidev) [inline ui/pages/home/beta/beta-home-footer.component.js:46]: typo here?

**Review by montelaidev — CHANGES_REQUESTED** [review]

File: ui/pages/home/beta/beta-home-footer.component.js (hunk 1 above)
Comment (gantunesr) [reply ui/pages/home/beta/beta-home-footer.component.js:46]: Debugging something unrelated. Pushed this by mistake

Comment (gantunesr) [reply ui/pages/home/beta/beta-home-footer.component.js]: Removed in [0be748e](https://github.com/MetaMask/metamask-extension/pull/21222/commits/0be748e254e8daf73c6e8101e9aa0572f5d323ee)

**Review by montelaidev — APPROVED** [review]
LGTM

**Review by danroc — APPROVED** [review]

**Pull request merged**